app/models/Restaurant.java

```diff
@@ -11,10 +13,11 @@
  */
 @Entity
 @Table(name = "abh_restaurant")
-public class Restaurant extends Model{
+public class Restaurant extends Model implements Validation{
     @Id
     @Column(name = "restaurantId", columnDefinition = "BIGINT")
     @JsonView(RestaurantViews.BasicDetails.class)
```

Comment:
Please, name classes in singular. In this case View and BasicDetail. Also, do you really need RestaurantView class or you can have global classes like: BasicView, FullView or something similar?